Docs 7741 clean up deploying to cloudhub hn (#344)
mulesoft/docs-runtime-manager: modules/ROOT/pages/cloudhub-fabric.adoc

> Use the drop-down menus next to *workers* to pick the amount and a size for the workers of your application and configure the computing power that you need.

Reviewer: ```suggestion
Use the drop-down menus next to *workers* to select the number and size of workers of your application and to configure the computing power that you need.
```

> Each application can be deployed with up to 8 workers of any kind, totaling up to 128 vCores. However, you may be limited to fewer vCores than those you need, based on how many are available in your subscription. See xref:deploying-to-cloudhub.adoc#worker-size-and-vcores[Worker Size and vCores] for information about deploying to multiple vCores.

Reviewer: I know that this might change the meaning you intend, so please look carefully and adjust as necessary:
```suggestion
Depending on your subscription, your application can include up to 8 workers of any kind, and your total applications can use up to 128 vCores. To ensure that you have sufficient resources, see xref:deploying-to-cloudhub.adoc#worker-size-and-vcores[Worker Size and vCores].
```

> Worker scale out also adds additional reliability. MuleSoft automatically distributes multiple workers for the same application across two or more data centers for maximum reliability.

Reviewer: ```suggestion
Worker scale out also adds additional reliability. Mule runtime engine (Mule) automatically distributes multiple workers for the same application across two or more data centers for maximum reliability.
```

* When you deploy an application to CloudHub for the first time using the Runtime Manager console
* By accessing the *Deployment* tab in the Runtime Manager console for a previously deployed application

> Next to *Workers*, select options from the drop-down menus to define the number and type of workers assigned to your application. See xref:deploying-to-cloudhub.adoc#worker-size-and-vcores[Worker Size and vCores] for more information about deploying to multiple workers.

Reviewer: Is this supposed to be a step? If so, format:
```suggestion
. Next to *Workers*, select options from the drop-down menus to define the number and type of workers assigned to your application. +

See xref:deploying-to-cloudhub.adoc#worker-size-and-vcores[Worker Size and vCores] for more information about deploying to multiple workers.
```

> Click an application to see the overview and click *Manage Application*. Click *Settings* and click the *Persistent Queues* checkbox to enable queue persistence.

Reviewer: Mos' def steps, and more than one....

Author: This topic didn't get touched except to fix xrefs... Will catch it next time.
